nemo/collections/common/tokenizers/huggingface/auto_tokenizer.py

```diff
@@ -203,7 +172,7 @@ def text_to_tokens(self, text):
 
     def tokens_to_text(self, tokens):
         text = self.tokenizer.convert_tokens_to_string(tokens)
-        return remove_spaces(handle_quotes(text.strip()))
+        return text
```

Comment:
High-level question about changes to this entire file - are we sure it doesn't break things for others using HF tokenizers?

Reply:
The initial reason for all this post-processing was to correctly process all the punctuation after bert-base-uncased detokenization 2 years ago when we tried to fine-tune it on En-De NMT. After that the tokenization/detokenization in HF improved significantly.

Reply:
I see, makes sense. Thanks for clarifying!